ec2/sc-ec2-windows-jumpcloud.yaml

```diff
@@ -32,20 +35,25 @@ Parameters:
       - t3.large
       - t3.xlarge
     Default: t3.micro
-    Description: Amazon EC2 Windows Instance Type
+    Description: Amazon EC2 Instance Type
     Type: String
   KeyPair:
-    Description: Name of existing EC2 key pair for Windows Instances
+    Description: Name of existing EC2 key pair
     Type: AWS::EC2::KeyPair::KeyName
   AmiId:
-    Type: AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>
+    Type: AWS::EC2::Image::Id
     Default: ami-074545a6fb2313e2c
   VolumeSize:
     Description: The EC2 volume size (in GB)
     Type: Number
     Default: 30
     MinValue: 10
     MaxValue: 2000
+  OwnerEmail:
```

Comment:
isn't this already a required tag in the SC?  would SC end user need to set this twice?

Reply:
That's right...but it's needed for 01_associate_jc_systems. Also doing it that way (requiring twice) in the linux template. I could rewrite these to pull OwnerEmail from tags as with TagRootVolume though.

Reply:
only setting it once would be desirable but that can be in a follow on PR if you want

Reply:
only setting it once would be desirable but that can be in a follow on PR if you want

Reply:
Agree will work on that next.